apps/studio/.env.test

```diff
@@ -10,26 +10,16 @@ NEXT_PUBLIC_APP_ENV='test'
 
 # Used for E2E testing
 DATABASE_URL=postgres://root:root@localhost:5432/test
-
+CI=false
```

Comment:
question: eh, why do we need to disable and are there any downstream impact?

Reply:
this was a mistake, lemme set to `true`